requirements/runtime_cuda.txt

```diff
@@ -5,7 +5,7 @@ fire
 mmengine-lite
 numpy<2.0.0
 openai
-outlines
+outlines<0.1.0
```

Comment:
How about letting pytorch engine use `xgrammar` instread of `outlines`?
It's uncessary to keep both guided decoding backends